docs/SwipeListView.md

```diff
@@ -6,6 +7,8 @@ A List that renders `<SwipeRow />`s
 
 | Prop | Notes | Type | Signature (func) | Default |
 |---|---|---|---|---|
+| `data` | List of objects to be passed into the `renderItem` and `renderHiddenItem` functions. Each item must include a unique `key` property or `keyExtractor` must be implemented to ensure full functionality. | `array` || 
+| `keyExtractor` | Function to generate `key` value for each row in the list. | `func` |`(item, index) => {}`| 
```

Comment:
I need some advice here on the signature of this function. Wasn't sure how to mimic the format you had for the other ones